dkms.in

```diff
@@ -1741,7 +1744,8 @@ do_status() {
                 echo
                 ;;
         esac
-    done < <(module_status "$@")
```

Comment:
Won't this mask away the warnings, when `dkms status` is used?

Can we bribe you to add some tests - it will make it clearer what the expected output ought to be. Thanks o/